src/main/java/globalquake/ui/globalquake/GlobalQuakePanel.java

```diff
@@ -110,7 +111,16 @@ private void drawTexts(Graphics2D g) {
         g.drawString(str, getWidth() - g.getFontMetrics().stringWidth(str) - 6, getHeight() - 9);
 
         List<String> settingsStrings = new ArrayList<>();
-        settingsStrings.add("Sound Alarms: %s (S)".formatted(Settings.enableSound ? "Enabled" : "Disabled"));
+
+        //If sound is not available, set a special message
+        if(!Sounds.soundsAvailable)
+        {
+            settingsStrings.add("Sound Alarm unavailable: See log");
```

Comment:
Just a detail.. but I would let the message say "Sound Alarms unavailable" as the user would already see the error popup during the loading process